client/state/plugins/wporg/actions.js

```diff
@@ -18,34 +15,32 @@ import { WPORG_PLUGIN_DATA_RECEIVE, FETCH_WPORG_PLUGIN_DATA } from 'state/action
 /**
  *  Local variables;
  */
-let _fetching = {};
-
-export default {
-	fetchPluginData: function( pluginSlug ) {
-		return dispatch => {
-			if ( _fetching[ pluginSlug ] ) {
-				return;
-			}
-			_fetching[ pluginSlug ] = true;
-
-			setTimeout( () => {
-				dispatch( {
-					type: FETCH_WPORG_PLUGIN_DATA,
-					pluginSlug: pluginSlug,
-				} );
-			}, 1 );
+const _fetching = {};
+
+export function fetchPluginData( pluginSlug ) {
+	return dispatch => {
+		if ( _fetching[ pluginSlug ] ) {
+			return;
+		}
+		_fetching[ pluginSlug ] = true;
+
+		setTimeout( () => {
+			dispatch( {
+				type: FETCH_WPORG_PLUGIN_DATA,
+				pluginSlug: pluginSlug,
+			} );
+		}, 1 );
```

Comment:
might want to switch this to `Promise.resolve().then( ... )` given recent finding with setTimeout with a small delay getting delayed weirdly. But if we do that, we probably want a generic `defer` type thing, which lodash already has.

Reply:
yeah I just left this for now - didn't want to open that can of worms